pkgs/development/tools/azure-static-sites-client/default.nix

```diff
@@ -0,0 +1,116 @@
+{ lib
+, stdenv
+, fetchurl
+, autoPatchelfHook
+, makeWrapper
+, curl
+, icu70
+, libkrb5
+, lttng-ust
+, openssl_1_1
+, zlib
+, azure-static-sites-client
+  # "latest", "stable" or "backup"
+, versionFlavor ? "stable"
+}:
+let
+  icu = icu70;
```

Comment:
This should be done in top-level or not at all.

Reply:
Remove the statement to use `icu70` directly.